apps/web/vibes/2px/components/date-picker/index.tsx

```diff
@@ -0,0 +1,89 @@
+'use client'
+
+import { useRef, useState } from 'react'
+
+import { DateValue, parseAbsoluteToLocal, parseDate } from '@internationalized/date'
+import { DateInput, DateInputProps, DateInputValue } from '@nextui-org/date-input'
+
+import { cn } from '@/lib/utils'
+import Calendar from '@/vibes/2px/components/calendar'
+import { CalendarIcon } from '@/vibes/2px/components/icons/CalendarIcon'
+
+import useOnClickOutside from './useOnClickOutside'
+
+interface Props extends Omit<DateInputProps, 'variant' | 'isDisabled'> {
+  variant?: 'default' | 'success' | 'error'
+  disabled?: boolean
+  onChange?: ((value?: DateValue) => void) | undefined
+}
+
+export default function DatePicker({
+  variant,
+  disabled: isDisabled,
+  value,
+  onChange,
+  ...props
+}: Props) {
+  const [selectedDate, setSelectedDate] = useState<DateInputValue | undefined | null>(value)
+  const [calendarOpen, setCalendarOpen] = useState(false)
+  const calendarRef = useRef<HTMLDivElement>(null)
+
+  useOnClickOutside(calendarRef, () => setCalendarOpen(false))
+
+  const handleCalendarSelect = (date?: Date) => {
+    const formattedDate = date?.toISOString()
+    ;(onChange || setSelectedDate)(
+      formattedDate
+        ? parseDate(parseAbsoluteToLocal(formattedDate).toString().split('T')[0])
```

Comment:
Whenever I select a date in the calendar, the day before gets painted, although the right date is inserted in the input. 

I think this might be caused by the timezone difference. 

![image](https://github.com/user-attachments/assets/f8ecdbf8-9eda-4f77-b24d-49cca6009f80)